plugins/generator-1.21.1/neoforge-1.21.1/procedures/world_entity_inrange_exists.java.ftl

```diff
@@ -1,3 +1,3 @@
 (!world.getEntitiesOfClass(${generator.map(field$entity, "entities", 0)}.class,
-	AABB.ofSize(new Vec3(${input$x}, ${input$y}, ${input$z}), ${input$range}, ${input$range}, ${input$range}), e -> true)
+	new AABB(Vec3.ZERO, Vec3.ZERO).move(new Vec3(${input$x}, ${input$y}, ${input$z})).inflate(${input$range} / 2d), e -> true)

```

Comment:
Is this really the same with half the range? Old one was likely from the corner, now it is from center.

Also is this really cleaner code?